src/main/java/uk/gov/companieshouse/insolvency/data/service/InsolvencyServiceImpl.java

```diff
@@ -130,7 +130,8 @@ public void deleteInsolvency(String contextId, String companyNumber) {
                     "Company insolvency delete called for company number %s",
                     companyNumber));
 
-            insolvencyApiService.invokeChsKafkaApi(contextId, companyNumber, "deleted");
+            insolvencyApiService.invokeChsKafkaApi(contextId, insolvencyDocumentOptional.get(),
```

Comment:
Probably better to avoid using .get() on an Option, maybe use .map() on option

Reply:
get should be ok, We are not transforming the object to other form, Moreover in the above step we are checking if not present.